packages/core/src/types/metadata.ts

```diff
@@ -11,27 +26,26 @@ export type ComponentMetadata = {
   [index: string]: any;
   httpRequests?: Record<string, string>;
   options?: TargetOptions;
-  angular?: {
-    /* Mitosis uses `attr.XXX` as default see https://angular.io/guide/attribute-binding. 
-    If you want to skip some you can use the 'nativeAttributes'. */
-    nativeAttributes?: string[];
-    /* Overwrite default selector for component. Default will be kebab case (MyComponent -> my-component) */
-    selector?: string;
-  };
-  qwik?: {
-    component?: {
-      isLight?: boolean;
-    };
-    setUseStoreFirst?: boolean;
-    hasDeepStore?: boolean;
-    mutable?: string[];
-    imports?: Dictionary<string>;
-    replace?: Record<string, string>;
-  };
-  solid?: {
-    state?: Dictionary<SolidState>;
-  };
-  rsc?: {
-    componentType?: 'client' | 'server';
-  };
+  tagName?: string; // Deprecated: Use this for web-components to change the tagName
+  forwardRef?: string; // Deprecated: Use this for react forwardRef
+  isAttachedToShadowDom?: boolean; // Enables shadowDom for web-components
```

Comment:
if you use JSDoc @deprecated syntax, the comments will flow to the end-user when using `useMetadata`:

```suggestion
  /** @deprecated Use this for web-components to change the tagName  */
  tagName?: string;
  /** @deprecated Use this for react forwardRef */
  forwardRef?: string;
  /** Enables shadowDom for web-components */
  isAttachedToShadowDom?: boolean;
```